PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java

```diff
@@ -270,7 +270,7 @@ public String promptForOpenSSLPath() {
     }
 
     @Override
-    public void openView(String sid, String id) {
+    public void openRedisPropertyView(String sid, String id) {
```

Comment:
It's dedicated now, will try to generalize it when we need to show more kind of property views